frontend/src/app/leaderboard/topStats.tsx

```
@@ -0,0 +1,64 @@
+          <div className="stat-title text-xs md:text-base">Total Tasks</div>
+          <div className="stat-value text-base md:text-3xl text-primary">100</div>
+        </div>
+
+        <div className="stat">
+          <div className="stat-figure hidden md:block text-secondary">
+          <svg
+              xmlns="http://www.w3.org/2000/svg"
+              fill="none"
+              viewBox="0 0 24 24"
+              className="inline-block h-8 w-8 stroke-current"
+            >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)